### PR TITLE
cmr-7144 - Updating API doc and adding a test

### PR DESCRIPTION
**Pull request by jceaser**
updating the documentation to mention URL encoding and also adding a test for more code coverage which was in a ticket that got closed.

### CODE REVIEW

**Review by mschmele — APPROVED** [review]

File: ingest-app/test/cmr/ingest/test/services/jobs.clj [hunk 1]
```diff
@@ -1,51 +1,96 @@
...
 (deftest email-granule-url-list-test
- "This tests the utility function that unpacts a list of urls and turns it into markdown"
- (let [actual (jobs/email-granule-url-list '("https://cmr.link/g1"
-                                             "https://cmr.link/g2"
-                                             "https://cmr.link/g3"))
-       expected (str "* [https://cmr.link/g1](https://cmr.link/g1)\n"
-                     "* [https://cmr.link/g2](https://cmr.link/g2)\n"
-                     "* [https://cmr.link/g3](https://cmr.link/g3)")]
-  (is (= expected actual))))
+  "This tests the utility function that unpacts a list of urls and turns it into markdown"
```
Comment (mitchstartzel) [inline ingest-app/test/cmr/ingest/test/services/jobs.clj:15]: unpacks?

File: ingest-app/test/cmr/ingest/test/services/jobs.clj [hunk 2]
```diff
@@ -1,51 +1,96 @@
...
+                "<ul><li><a href='https://cmr.link/g1'>https://cmr.link/g1</a></li><li>"
+                "<a href='https://cmr.link/g2'>https://cmr.link/g2</a></li><li>"
+                "<a href='https://cmr.link/g3'>https://cmr.link/g3</a></li></ul>"
+                "<p>To unsubscribe from these notifications, or if you have any questions, "
+                "please contact us at <a href='mailto:cmr-support@earthdata.nasa.gov'>"
+                "cmr-support@earthdata.nasa.gov</a>.</p>")
+           (:content (first (:body actual)))))))
+
+(deftest subscription->time-constraint-test
+  "Test the subscription->time-constraint function as it is criticle for internal
```
Comment (mitchstartzel) [inline ingest-app/test/cmr/ingest/test/services/jobs.clj:55]: ~~criticle~~ critical

**Review by mitchstartzel — APPROVED** [review]

File: ingest-app/test/cmr/ingest/test/services/jobs.clj (hunk 1 above)
Comment (jceaser) [reply ingest-app/test/cmr/ingest/test/services/jobs.clj:15]: sure, lets go with that (not my text)

File: ingest-app/test/cmr/ingest/test/services/jobs.clj (hunk 2 above)
Comment (jceaser) [reply ingest-app/test/cmr/ingest/test/services/jobs.clj:55]: this was me

**Pull request merged**